### PR TITLE
Add CAP text ETL and UI

### PR DESCRIPTION
**Pull request by danielnaab**
Implements issue(s) #27

This adds an extra ETL for the Corrective Action Plan text, which is the last AC item on #27

Changes proposed in this pull request:
- Add CAPText model
- Add CAPText download/ETL
- Add CAPText to search results table
- Add a couple indices that were missed previously

### CODE REVIEW

File: distiller/data/etls/load_dumps.py [hunk 1]
```diff
@@ -126,12 +124,16 @@ def update_table(
...
+def _sanitize_row(row, *, field_mapping, sanitizers, **_kwargs):
     sanitized_row = {}
     for csv_column_name, model_field_name in field_mapping.items():
         # These are fixedwidth CSVs, so strip off excess whitespace and handle
         # NULL values.
-        value = row[csv_column_name].strip() or None
+        try:
+            value = row[csv_column_name].strip() or None
+        except:
+            import pdb; pdb.set_trace()  #pylint: disable=C0321
```
Comment (cantsin) [inline distiller/data/etls/load_dumps.py]: Seems like leftover debug code -- suggest removing

File: distiller/settings/development.py [hunk 2]
```diff
@@ -10,3 +10,5 @@
 DEBUG = True
 
 SECRET_KEY = 'SECRET'
+
+LOAD_TABLE_ROOT = str(PROJECT_ROOT / 'imports')
```
Comment (cantsin) [inline distiller/settings/development.py]: `os.path.join`? Actually, not sure what this is for.

Comment (danielnaab) [reply distiller/settings/development.py]: This is a default download dir, if you run the download task locally. `PROJECT_ROOT` is a `pathlib.Path` object so `os.path.join` isn't required.

File: distiller/data/etls/load_dumps.py (hunk 1 above)
Comment (danielnaab) [reply distiller/data/etls/load_dumps.py]: Yep oops, thanks

**Comment by cantsin** [comment]
LGTM :raised_hands:

**Pull request merged**